Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/User/User/ByUser.tsx

```diff
@@ -156,7 +156,7 @@ const ByUser: Application.Types.iByComponent = (props) => {
                     if (grpIndex < 0)
                         return () => { }
 
-                    setOptions(valueListItems.filter(v => v.GroupID === valueListGroups[grpIndex].ID).map(item => ({ Value: item.ID.toString(), Label: item.Value })));
+                    setOptions(valueListItems.filter(v => v.GroupID === valueListGroups[grpIndex].ID).map(item => ({ Value: item.Value, Label: item?.AltValue ?? item.Value })));
```

Comment:
you can skip the ? in `item?,AltValue`  unless there is a possibility that `item` is null. If that is a possibility this will break in `item.Value` and would need to be `item?.Value`